Source/Core/InputCommon/ControllerEmu/ControlGroup/Cursor.h

```diff
@@ -47,22 +54,21 @@ class Cursor : public ReshapableInput
   static constexpr int AUTO_HIDE_MS = 2500;
   static constexpr double AUTO_HIDE_DEADZONE = 0.001;
 
-  // Not adjusted by width/height/center:
-  StateData m_state;
-
-  // Adjusted:
-  StateData m_prev_result;
+  // Not adjusted by width/height/center.
```

Comment:
Instead of `Cursor` holding dual state (which is needed) it would be cleaner if there was a struct to pass along to `GetState`. Mappping indicators and WiimoteEmu would each have their own instance of course.
This could be done later though. Your fix works for now.

Reply:
Yeah I know it's not the best way of implementing it but it's fine. If more is needed it can be done later.